.github/workflows/docs.yaml

```diff
@@ -22,10 +22,19 @@ jobs:
         with:
           go-version: 1.18.x
 
-      - uses: actions/cache@v1
+      - uses: actions/cache@v3
         with:
           path: ~/go/pkg/mod
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - uses: actions/cache@v3
+        with:
+          path: .mdoxcache
+          key: ${{ runner.os }}-mdox-${{ hashFiles('docs/**/*.md', 'examples/**/*.md', 'mixin/**/*.md', '*.md') }}
+          restore-keys: |
+            ${{ runner.os }}-mdox-
```

Comment:
I wonder if a maintainer could check if this is actually in cache now with https://docs.github.com/en/rest/actions/cache#list-github-actions-caches-for-a-repository. Would be neat to check beforehand!